activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -1,32 +1,48 @@
 module ActiveSupport
   module Testing
     class SimpleStubs # :nodoc:
-      Stub = Struct.new(:object, :method_name, :original_method)
+      Stub = Struct.new(:object, :method_name, :original_method, :return_value)
 
       def initialize
-        @stubs = {}
+        @stubs = Hash.new { |h,k| h[k] = [] }
```

Comment:
Would flipping the relationship clarify the intent? Found having stubs be per method and then the stub levels on each method hard to grasp and read.